### ISSUE TITLE
how to use

### ISSUE THREAD

**Issue by hengkyhks** [Naeaerc20/BerachainTestnet#1]
how to use, ? i cant run this script

can you guide me

**Comment by Naeaerc20** [comment]
Yes sure, main file which is "index.js" isn't built yet cause I'm building each dapp action so as is undone yet. Advisable is to open "actions" or "scripts" or "route" folders and on each you'll find same structure ABI.js, index.js and "scripts" so only run "index.js" on each dapp you want to use and it will work example...

cd BerachainTestnet
cd actions
cd BeraSwap
node index.js

don't forget to run "npm install" on main directory "BerachainTestnet" so each thing can work well and set your wallets on wallets.json

**Comment by hengkyhks** [comment]
node index.js on beraswap is null
